app/popup.html

```diff
@@ -53,16 +54,17 @@ <h1 class="text-center text-lg font-light">
           />
         </svg>
       </button>
-      <p class="text-gray-500 text-xs text-center">
+      <p class="m-1 text-gray-500 text-xs text-center">
         Reload the app when you add a new website
       </p>
     </div>
-
-    <script src="popup.js"></script>
+    <script type="module" src="./helpers/get-color-levels.js"></script>
```

Comment:
Hi.

Importing get-color-levels and get-time-left is useless because they will be imported automatically through ES Modules in popup.js. You can delete these lines.

Reply:
Hi ! 
I tested as you say and would not work. Files are correctly imported in background.js but not in popup.js
Es6 `import` will not work for `popup.js` outside of modules. Using `import` without these scripts will make popup.js fail...

https://medium.com/front-end-weekly/es6-modules-in-chrome-extensions-an-introduction-313b3fce955b

Reply:
You have to keep the `type="module"` on popup.js but you can delete the lines where you import get-color-levels.js and get-time-left because popup.js will import them for you. I just tried it on a clone of your repo and it works. We can organize a small meeting if it is not clear! :)

![image](https://user-images.githubusercontent.com/35616365/101349430-a386d980-388d-11eb-9acf-b15c37fb0cfe.png)

Reply:
Ok cool ! I'll try it like this !